job-tracker/src/components/EditJob/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+export class EditJob extends Component {
+  render() {
+    const { application, closeEdit } = this.props.job;
+
+    return (
+      <div className="edit-job">
+        <div className="container">
+          <h3 className="edit-job-title">Edit:</h3>
+
+          <button onClick={() => closeEdit}>Close Edit</button>
```

Comment:
Use a word like `Cancel` - words `Close Edit` put together like this can confuse, since they usually mean 2 separate actions.

Reply:
👍 Good catch.
I was scrambling and running out of time here so was typing nonsensical names not really thinking. LOL